### DISCUSSION TITLE
ESPresense flash to an ESP32-C3-12F?

### DISCUSSION THREAD

**Discussion by greggotcher** [ESPresense/ESPresense#137] [Q&A] [answered]
Is it possible to flash ESPresense to an ESP32-C3-12F?  I just ordered a bunch of these to do some chip transplants into some devices I currently have.  I will have some left over and wondering if I can use these for ESPresense.  If it doesn't I will just have to order up some ESP32 D1 Minis.  Thanks for all of the great work.

**Comment by DTTerastar** [comment, marked as answer]
C3's are a totally different kind of processor.  It'll take a bit before they'll be supported.
